assets/stylesheets/forms.scss

```diff
@@ -7,12 +7,23 @@
 .btn .fa {
     padding-right: 3pt;
 }
-.form-control {
+.form-control,.form-control:focus {
```

Comment:
without comma

Reply:
Why? It should match both `.form-control` as well as `.form-control:focus`

Reply:
It it should match both a comma is correct. I'd still prefera whitespace after the comma.